requirements-build.txt

```diff
@@ -19,3 +19,12 @@ pylint==3.3.1
 
 # Testing API keys
 google-genai==1.28.0
+
+# News Sentiment Analysis
+newspaper3k==0.2.8
```

Comment:
These requirements are specific to the news sentiment analysis use case. Move them to a `requirements.txt` file in `coded_tools/news_sentiment_analysis`